### PR TITLE
feat: adds telemetry page

### PR DESCRIPTION
**Pull request by BRKalow**
fixes SDK-981

### CODE REVIEW

File: docs/components/clerk-provider.mdx [hunk 1]
```diff
@@ -108,6 +108,7 @@ ReactDOM.render(
...
 | `domain?`                 | `string \| ((url: URL) => boolean)`                                                   | This option sets the domain of the satellite application. If your application is a satellite application, this option is required.                                                                                         |
 | `signInUrl`               | `string`                                                                              | This url will be used for any redirects that might happen and needs to point to your primary application. This option is optional for production instances and required for development instances.                         |
+| `telemetry`               | `false \| { disabled?: boolean; debug?: boolean } \| undefined`                       | Controls whether or not Clerk will collect [telemetry data](/docs/telemetry).                                                                                                                                              |
```
Comment (alexisintech) [inline docs/components/clerk-provider.mdx:111]: Is this required or optional?

File: docs/references/javascript/clerk/clerk.mdx [hunk 2]
```diff
@@ -99,6 +99,7 @@ All props below are optional.
 | `allowedRedirectOrigins` | `Array<string \| RegExp> \| undefined` | Optional array of domains used to validate against the query param of an auth redirect.<br />If no match is made, the redirect is considered unsafe and the default redirect will be used with a warning passed to the console. |
 | `isInterstitial` | `boolean \| undefined` | Indicates that Clerk.js will be loaded from interstitial. Defaults to `false` |
 | `isSatellite` | `boolean \| ((url: URL) => boolean) \| undefined` | Clerk Flag for satellite apps. Experimental. |
+| `telemetry` | `false \| { disabled?: boolean; debug?: boolean } \| undefined` | Controls whether or not Clerk will collect [telemetry data](/docs/telemetry). |
```
Comment (alexisintech) [inline docs/references/javascript/clerk/clerk.mdx:102]: Is this required or optional?

File: docs/telemetry.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  payload: { component: 'SignIn', appearanceProp: false },
+}
+```
+
+- **event** — A unique identifier for the event type
+- **cv** — Clerk Version. the version of the core Clerk library in use
+- **sdk** — SDK. The Clerk SDK that is being used
+- **sdkv** — SDK Version. The version of the Clerk SDK
+- **pk** — Publishable Key. The instance's publishable key
+- **payload** — Each unique event can provide custom data as part of the payload. As seen above, for the `COMPONENT_MOUNTED` event we track the component name and additional data about prop usage
```
Comment (alexisintech) [inline docs/telemetry.mdx:51]:
```suggestion
- `event` — A unique identifier for the event type.
- `cv` — Clerk Version. The version of the core Clerk library in use.
- `sdk` — SDK. The Clerk SDK that is being used.
- `sdkv` — SDK Version. The version of the Clerk SDK.
- `pk` — Publishable Key. The instance's publishable key.
- `payload` — Each unique event can provide custom data as part of the payload. As seen above, for the `COMPONENT_MOUNTED` event we track the component name and additional data about prop usage.
```

File: docs/telemetry.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
+
+### Expo
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
```
Comment (alexisintech) [inline docs/telemetry.mdx:114]: Should this be in a `<CodeBlockTabs />` component?

**Review by alexisintech — CHANGES_REQUESTED** [review]

File: docs/components/clerk-provider.mdx (hunk 1 above)
Comment (BRKalow) [reply docs/components/clerk-provider.mdx:111]: Optional, good catch!

Comment (BRKalow) [inline docs/components/clerk-provider.mdx:111]:
```suggestion
| `telemetry?`               | `false \| { disabled?: boolean; debug?: boolean } \| undefined`                       | Controls whether or not Clerk will collect [telemetry data](/docs/telemetry).                                                                                                                                              |

```

File: docs/references/javascript/clerk/clerk.mdx (hunk 2 above)
Comment (BRKalow) [inline docs/references/javascript/clerk/clerk.mdx:102]:
```suggestion
| `telemetry?` | `false \| { disabled?: boolean; debug?: boolean } \| undefined` | Controls whether or not Clerk will collect [telemetry data](/docs/telemetry). |

```

File: docs/telemetry.mdx (hunk 3 above)
Comment (BRKalow) [reply docs/telemetry.mdx:51]: Thanks!

File: docs/telemetry.mdx (hunk 4 above)
Comment (BRKalow) [reply docs/telemetry.mdx:114]: @alexisintech I think for this situation I kinda prefer the non-tabs approach. IMO, it makes it easier to scan and quickly find the section for a specific framework. WDYT?

Screenshot 2023-11-20 at 1 23 11 PM

File: docs/telemetry.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+### Environment variables
+
+<Callout type="warning">
+  Note that the variable name might differ between frameworks. See the [framework specific instructions]() below.
+</Callout>
+
+For meta-framework SDKs, you can opt-out of telemetry collection by setting the environment variable: 
+
+```env filename=".env.local"
+CLERK_TELEMETRY_DISABLED=1`.
```
Comment (BRKalow) [inline docs/telemetry.mdx:74]:
```suggestion
CLERK_TELEMETRY_DISABLED=1
```

**Comment by alexisintech** [comment]
Where can this be placed in the navigation? Maybe under advanced-usage for now? Could you add this page to the manifest.json (but do not move its location. it's fine under /docs in order to have a flat url path of /docs/telemetry as this is what we are moving the rest of the pages towards)

**Comment by BRKalow** [comment]
@alexisintech I was considering not having it in the nav 👀. Maybe that's a bad idea though. More likely this is something that would be linked to in our footer, or somewhere else more global.

**Comment by rachelnabors** [comment]
One thought is to make a "Privacy" nav item under "Security" and place this there alongside any other privacy-related content and assurances we might provide (like recommendations around user impersonation). Seems like something you'd expect to find in an auth provider's docs! Short of that, agree with @BRKalow that the footer might make sense.

File: docs/telemetry.mdx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+Clerk takes data privacy and protection seriously. Telemetry data is most useful in aggregate form to gain product insights, and the raw data is only available to a small subset of Clerk employees.
+
+We will never share with or sell telemetry data to third parties. The data is used strictly to help improve the Clerk product.
+
+## How do I opt-out?
+
+### Environment variables
+
+<Callout type="warning">
+  Note that the variable name might differ between frameworks. See the [framework specific instructions]() below.
```
Comment (LekoArts) [inline docs/telemetry.mdx:68]: Anchor link is missing here

File: docs/telemetry.mdx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+<ClerkProvider telemetry={false} />
+```
+
+```tsx
+CLERK_TELEMETRY_DISABLED=1
+```
+
+```tsx
+GATSBY_CLERK_TELEMETRY_DISABLED=1
+```
```
Comment (LekoArts) [inline docs/telemetry.mdx:102]: Should these be `tsx` code blocks?

File: docs/references/javascript/clerk/clerk.mdx [hunk 8]
```diff
@@ -230,4 +232,4 @@ In addition to the methods listed above, the `Clerk` class also has the followin
 [user-ref]: /docs/references/javascript/user/user
 [organization-ref]: /docs/references/javascript/organization/organization
```
Comment (rachelnabors) [inline docs/references/javascript/clerk/clerk.mdx:233]: These links indicate we're using multiple reference links. As per [Styleguide 1.3, let's dedupe the reference links.](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#13-de-dupe-reference-links)

**Review by rachelnabors — CHANGES_REQUESTED** [review]
From a structural perspective, I would rename the "Security" section to "Security and Privacy" and add this there. What do you think, @alexisintech ?

File: docs/references/javascript/clerk/clerk.mdx (hunk 8 above)
Comment (alexisintech) [reply docs/references/javascript/clerk/clerk.mdx:233]: Also, I agree with a "Security and Privacy" section, and this page can go under that

Comment (rachelnabors) [reply docs/references/javascript/clerk/clerk.mdx:233]: It's not an urgent matter, but something we should apply moving forward when accepting PRs.

Comment (rachelnabors) [reply docs/references/javascript/clerk/clerk.mdx:233]: @BRKalow mind moving this to "Security" in the sidebar and renaming that item "Security and Privacy?"

**Review by alexisintech — APPROVED** [review]
😸💖

**Review by LekoArts — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]
Purrrrfect!

**Comment by alexisintech** [comment]
holding off on merging until v5 is released

**Comment by rachelnabors** [comment]
Why we waiting till v5? Let's give it a tag and put it into draft till then

**Pull request merged**